src/game_event.cpp

```diff
@@ -465,7 +465,7 @@ void Game_Event::UpdateNextMovementAction() {
 
 void Game_Event::SetMaxStopCountForRandom() {
 	auto st = GetMaxStopCountForStep(GetMoveFrequency());
-	st *= (Rand::GetRandomNumber(0, 3) + 3) / 5;
+	st *= (Rand::GetRandomNumber(0, 3) + 3) / 5.0;
```

Comment:
Can you test something for me?

Change this to:

```cpp
st *= Rand::GetRandomNumber(0, 3) + 3;
st /= 5;
```

Reply:
Guess the precedence is just wrong because of the "*=". Everything is divided by 5, not just the random number.

Reply:
The change works for me. Should I update this PR this way?

Reply:
Yes please as we try to be close to rpg_rt here

Reply:
Updated this PR.